common/lib/xmodule/xmodule/library_root_xblock.py

```diff
@@ -71,7 +79,7 @@ def render_children(self, context, fragment, can_reorder=False, can_add=False):
         for child_key in children_to_show:  # pylint: disable=E1101
             child = self.runtime.get_block(child_key)
             child_view_name = StudioEditableModule.get_preview_view_name(child)
-            rendered_child = self.runtime.render_child(child, child_view_name, context)
+            rendered_child = self.render_child(child, child_view_name, context)
```

Comment:
Reading these comments, @bradenmacdonald has convinced me. @Kelketek, did you try this approach?